pdfbox/src/main/java/org/apache/pdfbox/pdmodel/font/encoding/Encoding.java

```diff
@@ -159,23 +159,18 @@ public boolean contains(int code)
 
     /**
      * This will take a character code and get the name from the code.
-     * 
+     *
      * @param code character code
-     * @return PostScript glyph name
+     * @return PostScript glyph name, or ".notdef" when not found.
      */
     public String getName(int code)
     {
-       String name = codeToName.get(code);
-       if (name != null)
-       {
-          return name;
-       }
-       return ".notdef";
+       return codeToName.getOrDefault(code, ".notdef");
```

Comment:
getOrDefault() is available from Java 8 on, ok for PDFBox  3+.

Reply:
getOrDefault() is used in 2 other locations as well

Reply:
I've added the proposed change as well, see PDFBOX-5660

Reply:
> getOrDefault() is used in 2 other locations as well

Yes, I just made that comment to remind whoever integrates this change doesn't pull it into PDFBox 2. Many fixes and improvements are backported to the old release that still supports Java 6.